pkg/maya/cstorvolume/v1alpha1/kubernetes_test.go

```diff
@@ -21,7 +21,8 @@ import (
 	"github.com/pkg/errors"
 
 	apis "github.com/openebs/csi/pkg/apis/openebs.io/maya/v1alpha1"
-	"github.com/openebs/csi/pkg/generated/clientset/versioned"
+	client "github.com/openebs/csi/pkg/generated/clientset/maya/internalclientset"
+	clientset "github.com/openebs/csi/pkg/generated/clientset/maya/internalclientset"
```

Comment:
is this correct?

Reply:
its happened b/c we were using `clientset` as function args, fixed